app/analysis/index/template.hbs

```diff
@@ -305,6 +305,8 @@
             type="button"
             data-download-count="{{download.performCount}}"
             class="btn btn-primary {{if (and (eq download.last.url link.url) (eq download.last.params link.params) download.isRunning) 'loading'}}"
+            disabled={{exportLimitExceeded}}
+            title={{if exportLimitExceeded "Please use filters to reduce the amount of reports to export."}}
```

Comment:
Include the export maximum here so the user knows it

Reply:
@anehx done